lib/routes/theinitium/utils.ts

```diff
@@ -39,7 +39,6 @@ export const processFeed = async (model: string, ctx: Context) => {
     const key = {
         email: config.initium.username,
         password: config.initium.password,
-        iapReceipt: config.initium.iap_receipt,
```

Comment:
https://github.com/DIYgod/RSSHub/blob/0f01fe60cbc6fc57f8fa01e258267954b2110d1f/lib/routes/theinitium/utils.ts#L144
is still using key.iapReceipt

Please remove corresponding entries in `lib/config.ts` as well.

Reply:
Thanks, entry removed.